docs/learning/Bottlenecking.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: default
+title: Bottle Necking
+nav_exclude: false
+has_children: false
+parent: Learning
+search_exclude: false
+last_modified_date: 2023-02-27
+---
+
+# BottleNecking

```

Comment:
Is it Bottle necking, BottleNecking, or Bottlenecking? You should use one for everything.